integration_test/third_party_apps_data/applications/couchbase/debian_ubuntu/install

```diff
@@ -13,8 +13,11 @@ sudo dpkg -i ./couchbase-release-1.0-amd64.deb
 sudo apt-get update
 sudo apt-get install -y couchbase-server-community=7.1.0-2556-1
 
+# Wait for the app to boot
+sleep 60
```

Comment:
Adding a sleep to fix errors like:

```
debian 9 ERROR: Unable to connect to host at http://127.0.0.1:8091/: HTTPConnectionPool(host='127.0.0.1', port=8091): Max retries exceeded with url: /pools (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d23fd0a30>: Failed to establish a new connection: [Errno 111] Connection refused'))debian 9 ERROR: Unable to connect to host at http://127.0.0.1:8091/: HTTPConnectionPool(host='127.0.0.1', port=8091): Max retries exceeded with url: /pools (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1d23fd0a30>: Failed to establish a new connection: [Errno 111] Connection refused'))
```

Reply:
Do you think doing something similar like
```sh
timeout 60s bash <<EOF
wait_for_couchbase() {
    until curl localhost:8091 > /dev/null 2>&1
    do
        echo "Waiting for couchbase to start. . ."
        sleep "1"
    done
}

wait_for_couchbase

sleep 5

echo "couchbase started"
EOF
```

would be preferred?

Reply:
Yes! That looks even better! 

I was just able to comment that I'm not 100% sure whether `60s` is enough.

BTW I was not able to get everything work last night with this PR. I initially attempted to fix the tests and hoped that the implementation is not broken. But it took quite some time to get the tests running on Debian 9, and other distros are probably also broken and need more work. So I ended up manually checking the implementation (I don't remember exactly how long I waited locally before the app is up and accepting requests).

Later we rolled back, and I haven't come back to this afterwards. Thanks for addressing this.